gender_analysis/common.py

```diff
@@ -7,12 +7,23 @@
 import seaborn as sns
 
 from gender_analysis.pronouns import PronounSeries
+from gender_analysis.gender import Gender
 
 BASE_PATH = Path(os.path.abspath(os.path.dirname(__file__)))
 TEST_DATA_PATH = Path(BASE_PATH, 'testing', 'test_data')
-MASC_WORDS = PronounSeries('Masc', {'he', 'his', 'him', 'himself'})
-FEM_WORDS = PronounSeries('Fem', {'she', 'her', 'hers', 'herself'})
 
+# Common Pronoun Collections
+HE_SERIES = PronounSeries('Masc', {'he', 'his', 'him', 'himself'})
+SHE_SERIES = PronounSeries('Fem', {'she', 'her', 'hers', 'herself'})
+THEY_SERIES = PronounSeries('Andy', {'they', 'them', 'theirs', 'themself'})
+
+# Common Gender Collections
+MALE = Gender('Male', HE_SERIES)
+FEMALE = Gender('Female', SHE_SERIES)
+NEUTRAL = Gender('Neutral', THEY_SERIES)
```

Comment:
This is probably fine, but I wonder if there's a better phrasing than `NEUTRAL`? I don't have a good answer for this, unfortunately, but personally I'm wary of referring to this as a "neutral" option as opposed to "non-binary" for instance. With that said, the use of "non-binary" as an all-encompassing rule for this is probably not the best anyway...